app/models/external_work.rb

```diff
@@ -1,55 +1,57 @@
 include UrlHelpers
 class ExternalWork < ActiveRecord::Base
-  
   include Taggable
   include Bookmarkable
 
   attr_protected :summary_sanitizer_version
-  
-  has_many :related_works, :as => :parent  
+
+  has_many :related_works, as: :parent
 
   belongs_to :language
-  
-  scope :duplicate, :group => "url HAVING count(DISTINCT id) > 1"
+
+  scope :duplicate, group: "url HAVING count(DISTINCT id) > 1"
 
   AUTHOR_LENGTH_MAX = 500
-  
+
   validates_presence_of :title
-  validates_length_of :title, :minimum => ArchiveConfig.TITLE_MIN, 
-    :too_short=> ts("must be at least %{min} characters long.", :min => ArchiveConfig.TITLE_MIN)
-  validates_length_of :title, :maximum => ArchiveConfig.TITLE_MAX, 
-    :too_long=> ts("must be less than %{max} characters long.", :max => ArchiveConfig.TITLE_MAX)
-    
-  validates_length_of :summary, :allow_blank => true, :maximum => ArchiveConfig.SUMMARY_MAX, 
-    :too_long => ts("must be less than %{max} characters long.", :max => ArchiveConfig.SUMMARY_MAX)
-      
-  validates_presence_of :author, :message => ts('^Creator can\'t be blank')
-  validates_length_of :author, :maximum => AUTHOR_LENGTH_MAX, 
-    :too_long=> ts('^Creator must be less than %{max} characters long.', :max => AUTHOR_LENGTH_MAX)
+  validates_length_of :title, minimum: ArchiveConfig.TITLE_MIN,
+                              too_short: ts("must be at least %{min} characters long.",
+                                            min: ArchiveConfig.TITLE_MIN)
+  validates_length_of :title, maximum: ArchiveConfig.TITLE_MAX,
+                              too_long: ts("must be less than %{max} characters long.",
+                                           max: ArchiveConfig.TITLE_MAX)
+
+  validates_length_of :summary, allow_blank: true, maximum: ArchiveConfig.SUMMARY_MAX,
+                                too_long: ts("must be less than %{max} characters long.",
+                                             max: ArchiveConfig.SUMMARY_MAX)
+
+  validates_presence_of :author, message: ts('^Creator can\'t be blank')
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.